Makefile

```diff
@@ -2,7 +2,11 @@
 NUM_ADDONS=10
 NUM_THEMES=$(NUM_ADDONS)
 
-COMPOSE_PROJECT_NAME?=addonsserver
+# Get the name of the Makefile's directory for the docker container base name.
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
```

Comment:
(not a comment suggesting you should change anything, just me trying to read the code is all)